opt_einsum/paths.py

```diff
@@ -914,41 +997,27 @@ def __call__(self, inputs, output, size_dict, memory_limit=None):
                     # try to combine solutions from x[m] and x[n-m]
                     for m in range(1, n // 2 + 1):
                         for s1, (i1, cost1, cntrct1) in x[m].items():
-                            for s2, (i2, cost2, cntrct2) in x[n-m].items():
+                            for s2, (i2, cost2, cntrct2) in x[n - m].items():
 
                                 # only if s1 and s2 are disjoint
-                                if s1 & s2 == 0:
+                                if not s1 & s2:
```

Comment:
Can we collapse this and the below if to a single line to help with the indention?

Reply:
Ah yes nice.